chart/values.yaml

```diff
@@ -85,6 +83,26 @@ serviceWaiter:
   imageName: "service-waiter"
 tracing: {}
 authProviders: []
+# - id: "Public-GitHub"
+#   host: "github.com"
+#   type: "GitHub"
+#   oauth:
+#     clientId: "CLIENT_ID"
+#     clientSecret: "SECRET"
+#     callBackUrl: "https://gitpod.io/auth/github/callback"
+#     settingsUrl: "https://github.com/settings/connections/applications/CLIENT_ID"
+#   description: ""
+#   icon: ""
+# - id: "Public-GitLab"
+#   host: "gitlab.com"
+#   type: "GitLab"
+#   oauth:
+#     clientId: "CLIENT_ID"
+#     clientSecret: "SECRET"
+#     callBackUrl: "https://gitpod.io/auth/gitlab/callback"
+#     settingsUrl: "https://gitlab.com/profile/applications"
+#   description: ""
+#   icon: ""
```

Comment:
leftover?

If not, a comment would be nice to explain why this is there. E.g.
```YAML
# # Below are example authProvider configurations. Uncomment and complete values to use them.
# - id: ...
```

Reply:
Added comment.